queue.c

```diff
@@ -2723,6 +2727,9 @@ BaseType_t xQueueIsQueueFullFromISR( const QueueHandle_t xQueue )
     {
         UBaseType_t ux;
 
+        configASSERT( xQueue );

```

Comment:
This change will result in users with configASSERT enabled not being able to add a NULL queue handle to the queue registry.